Sources/Core/AWSSDKIdentity/InternalClients/InternalAWSCognitoIdentity/Sources/InternalAWSCognitoIdentity/CognitoIdentityClient.swift

```diff
@@ -21,6 +21,7 @@ import class Smithy.ContextBuilder
 import class SmithyHTTPAPI.HTTPRequest
 import class SmithyHTTPAPI.HTTPResponse
```

Comment:
This file & the next few below demonstrate the changes to code-generated clients.